app/addons/components/components/zenmodeoverlay.js

```diff
@@ -15,7 +15,7 @@ import app from "../../../app";
 import {CodeEditor} from './codeeditor';
 import {Tooltip, OverlayTrigger} from 'react-bootstrap';
 
-require('brace/theme/dawn');
+require('ace-builds/src-min-noconflict/theme-dawn');
```

Comment:
Dawn theme was not getting applied. I had to add `import 'ace-builds/css/theme/dawn.css';` here to make it work.

Reply:
done